Solutions/CohesitySecurity/Analytic Rules/RansomwareAlert.yaml

```diff
@@ -0,0 +1,76 @@
+id: 52a51512-5c38-4e0e-a741-aa2afe6c8d34
+name: Ransomware Alert
+description: Take custom fields in alert
+severity: High
+requiredDataConnectors: []
+queryFrequency: 5m
+queryPeriod: 5m
```

Comment:
This won't work with your outlier lookback

Reply:
just change them to be 1h and 14d